ras_bt_framework/behavior_utility/yaml_parser.py

```diff
@@ -5,6 +5,13 @@
 from ament_index_python import get_package_share_directory
 from ..managers.keywords_module_generator import KeywordModuleGenerator
 def read_yaml_to_pose_dict(path):
+
+     # Load lab_setup.yaml
+    with open("/ras_sim_lab/configs/lab_setup.yaml", 'r') as lab_file:
```

Comment:
path is being passed for the experiment and lab_setup is passed in hardcoded way for a different reason. implement the parsing in labloader class in or seperate locations into a different config file. Need a discussion